### DISCUSSION TITLE
How to configure arbitrary subdomain to be able to use in Vercel?

### DISCUSSION THREAD

**Discussion by j4nos** [vercel/vercel#8188] [Help] [answered]
I would use, redirect from tikex-dev.com subdomain to subpathes. How can I configure Vercel to do that?

like:

`https://turi-nyeremenyjatek-im2q.tikex-dev.com` -> `https://tikex-dev.com/turi-nyeremenyjatek/im2q`

I tried set a TXT wildcard domain:

[![enter image description here][1]][1]

[![enter image description here][2]][2]

  [1]: https://i.stack.imgur.com/eHXxJ.png
  [2]: https://i.stack.imgur.com/eHCiR.png

1. Do I need to select TXT record? Or CNAME, or any other?
2. Do I need to write wild card domain *.tikex-dev.com into the value or the name field?

---

I also tried following this tutorial and at step 3 I filled domain with`*.` prefix. Did not work, but now all the site is down. https://vercel.com/blog/wildcard-domains

**Comment by amyegan** [comment, marked as answer]
Hi @j4nos. You can set up a redirect in a vercel.json file at the root of your project https://vercel.com/docs/project-configuration#project-configuration/redirects. 

If you're using Next.js, you can set it in your next.config.js instead https://nextjs.org/docs/api-reference/next.config.js/redirects
